src/Sarif/Writers/ConsoleLogger.cs

```diff
@@ -192,18 +198,12 @@ private void WriteToConsole(Notification notification)
                 // This notification type is optionally emitted.
                 case FailureLevel.None:
                 case FailureLevel.Note:
-                    if (Verbose)
-                    {
-                        WriteLineToConsole(FormatNotificationMessage(notification, _toolName));
-                    }
-                    break;
-
-                // These notification types are always emitted.
-                case FailureLevel.Error:
                 case FailureLevel.Warning:
                     WriteLineToConsole(FormatNotificationMessage(notification, _toolName));
                     break;
-
+                case FailureLevel.Error:
+                    WriteLineToConsole(FormatNotificationMessage(notification, _toolName), true);
```

Comment:
provide the name for this parameter? generally label a literal arg value.

Reply:
Done.

---
In reply to: [559735385](https://github.com/microsoft/sarif-sdk/pull/2246#discussion_r559735385) [](ancestors = 559735385)